recipes/larch-phylo/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "larch" %}
+{% set package_name = "larch-phylo" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ package_name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/matsengrp/larch.git
+  git_rev: conda-forge
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - yasm
+    - automake
+    - autoconf
+
+  host:
+    - libtool
+    - zlib
+    - openmpi ==5.0.8
+    - openmpi-mpicc ==5.0.8
+    - openmpi-mpicxx ==5.0.8
+    - libboost-devel ==1.85.0
+    - protobuf ==4.25.2
+    - libprotobuf ==4.25.2
+
+  run:
+    - openmpi ==5.0.8
+
+test:
+  commands:
+    - larch-usher --version
+    - larch-dagutil --version
+    - larch-dag2dot --version
+
+about:
+  home: https://github.com/matsengrp/larch
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  license_url: https://opensource.org/license/mit
+  summary: 'Phylogenetic tool for exploring tree space of mutation annotated DAG/trees through SPR moves'
+  description: |
+    Phylogenetic tool for exploring tree space of mutation annotated DAG/trees through SPR moves.
+  doc_url: https://github.com/matsengrp/larch/blob/main/README.md
+  dev_url: https://github.com/matsengrp/larch
+
+extra:
+  recipe-maintainers:
```

```suggestion
extra:
  feedstock-name: some-name-here
  recipe-maintainers:
```
Please use the feedstock-name extra to set the name of your feedstock since it seems to be ambigious.